### PR TITLE
Fill random docs

### PR DESCRIPTION
**Pull request by KoBeWi**

### CODE REVIEW

File: doc/classes/@GlobalScope.xml [hunk 1]
```diff
@@ -1377,8 +1378,10 @@
 			General horizontal alignment, usually used for [Separator], [ScrollBar], [Slider], etc.
 		</constant>
 		<constant name="CLOCKWISE" value="0" enum="ClockDirection">
+			Clockiwse rotation. Used by some methods (e.g. [method Image.rotate_90]).
```
Comment (RedMser) [inline doc/classes/@GlobalScope.xml:1381]:
```suggestion
			Clockwise rotation. Used by some methods (e.g. [method Image.rotate_90]).
```

File: doc/classes/ColorPicker.xml [hunk 2]
```diff
@@ -133,8 +134,10 @@
 			The indicator used to signalize that the color value is outside the 0-1 range.
 		</theme_item>
 		<theme_item name="picker_cursor" data_type="icon" type="Texture2D">
+			The image displayed over
```
Comment (RedMser) [inline doc/classes/ColorPicker.xml:137]: Seems unfinished...

File: doc/classes/Engine.xml [hunk 3]
```diff
@@ -163,6 +165,7 @@
 		<method name="get_singleton_list" qualifiers="const">
 			<return type="PackedStringArray" />
 			<description>
+				Returns a list of available singletons.
```
Comment (RedMser) [inline doc/classes/Engine.xml:168]: Would be interesting to note whether this only includes built-in or also user-defined singletons.

Comment (KoBeWi) [reply doc/classes/Engine.xml]: Yeah, it includes both, so no point in mentioning it. I only clarified user singletons where relevant.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you!

**Pull request merged**